tests/test.cpp

```diff
@@ -1667,6 +1667,13 @@ void ErrorTest() {
   TestError("enum X:byte { Y, Y }", "value already");
   TestError("enum X:byte { Y=2, Z=2 }", "unique");
   TestError("table X { Y:int; } table X {", "datatype already");
+  TestError("table X { A:bool; } table Y { } union X {",
```

Comment:
This seems like an incomplete schema definition. will that have any effect on the tests?

Reply:
It is incomplete but many of them are, I believe to check that the error is thrown at that point when parsing the schema.

Reply:
Acknowledge.